locallibrary/catalog/views.py

```diff
@@ -1,5 +1,7 @@
 from django.shortcuts import render
 from django.views import generic
+from django.contrib.auth.mixins import LoginRequiredMixin
+from django.contrib.auth.mixins import PermissionRequiredMixin
```

Comment:
Gom lại trên 1 dòng thôi nhé
```suggestion
from django.contrib.auth.mixins import LoginRequiredMixin, PermissionRequiredMixin
```